src/ci/bin/testMetadataComparisonPython.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+# import in shellcheck / CI / IntelliJ compatible ways
+# shellcheck source=/dev/null
+source "${BASH_SOURCE%/*}/test.inc.sh" || source test.inc.sh
+
+cromwell::build::setup_common_environment
+
+build_root="$( dirname "${BASH_SOURCE[0]}" )/../../.."
```

Comment:
doesn't a variable like this get set by the `cromwell::build::setup_common_environment` command above?